package.json

```diff
@@ -320,6 +320,7 @@
     "supports-color": "3.1.2"
   },
   "devDependencies": {
+    "@coderbyheart/karma-sauce-launcher": "coderbyheart/karma-sauce-launcher#5259942cd6d40090eaa13ceeef5b0b8738c7710f",
```

Comment:
`karma-sauce-launcher` was a misnomer.  npm@3 and npm@4 install this anyway, but npm@5 does not; see npm/npm#17193.

I'm using the changeset SHA here to avoid pulling down any subsequent commits to `master`, since this is someone else's repo.

I have another branch wherein the Mocha org owns a fork, so we have complete control over how it behaves -- and what upgrades it receives.

(This was done in the first place because the original package is not-very-well maintained)